### ISSUE TITLE
VMAP is not supported with live source stream

### ISSUE THREAD

**Issue by kslimani** [kslimani/clappr-ima-plugin#2]
Workaround: do not use VMAP VAST tag with live source.

**Comment by aFilipMalinowski** [comment]
I did not observe any issues with live streams using VMAP.

**Comment by aFilipMalinowski** [comment]
Sometimes however I experienced some "stability" errors. Like:
- midroll adbreak coming immediately after preroll adbreak
- no autoplay after adbreak ended

**Comment by kslimani** [comment]
@aFilipMalinowski yes, using midroll with live stream source may lead to unexpected behaviour as it is internally handled by IMA SDK.
